src/pages/VerdictPage.tsx

```diff
@@ -157,7 +156,7 @@ export default function VerdictPage() {
               ? memoryKilobytes + ' KB'
               : memoryMegaBytes + ' MB'
 
-          const code = `\`\`\`${data.language}\n${atob(data.code)}\n\`\`\``
+          const code = `\`\`\`${data.language}\n${atob(data.code)}\`\`\``
```

Comment:
Why remove the new line?

Reply:
Adding an additional newline after the code is finished doesn't make sense. I want the code displayed to reflect the actual file submitted.